tests/unit/measures/test_dummy_data.py

```diff
@@ -102,7 +102,8 @@ def test_configured_population_size(legacy):
         intervals=years(1).starting_on("2020-01-01"),
     )
 
-    measures.configure_dummy_data(population_size=10, legacy=legacy)
+    measures.configure_dummy_data(population_size=99, legacy=legacy, timeout=123)
```

Comment:
I just noticed that it was setting it to the default value (or what is now the default, maybe it wasn't when the test was written) and therefore this would pass even if the argument was completely ignored.